src/mmcmdl.c

```diff
@@ -2004,7 +2004,7 @@ void parseCommandLine(vstring line) {
       }
       /* If character is comment, we're done */
       if (p == 0 && instr(1,tokenComment,chr(line[p]))) {
-        break;
+        goto parseCommandLine_ret;
```

Comment:
continue (line 2003) and break are usual choices in a loop, goto is of ill repute

Reply:
The `continue` is fine, but the `break` can't stay once the `switch` is introduced because it will target the switch instead of the loop. I would use labeled break if C had that, but it doesn't and `goto` is the next best thing.